src/library/dao/directorydao.cpp

```diff
@@ -149,32 +150,35 @@ QList<RelocatedTrack> DirectoryDAO::relocateDirectory(
     // mysterious ways for example if a track in the oldFolder also has a zombie
     // track location in newFolder then the replace query will fail because the
     // location column becomes non-unique.
+    const QString oldFolderLocation = mixxx::FileInfo(oldFolder).location();
```

Comment:
I think it is not a good idea to mess with the old location.
We should take literally the string that is stored, to make the following statements work.

I think the same is true for the new location.
Sanitation should happen at a single place, where the user selects the new location not here in the DAO code.

Reply:
This contradicts with the transformation `QDir(oldFolder).absolutePath()` that is used in the existing code.